src/ui/app/pages/send.jsx

```diff
@@ -982,11 +972,12 @@ const AddressPopup = ({
         };
       }
     }
+
     return addr;
   };
 
   const handleInputDebounced = useConstant(() =>
-    debouncePromise(handleInput, 300)
+    debouncePromise(latest(handleInput), 300)
```

Comment:
I suggest increasing this, because ada handle api is rate limited. 700 is the value used in Lace
```suggestion
    debouncePromise(latest(handleInput), 700)
```